Automattic/wp-calypso
# Domains: Add functionality to options menu items

#### Changes proposed in this Pull Request

This adds functionality to "Renew now" button. The auto-renew button has been removed and will be addressed in a separate PR.
 
#### Testing instructions

- Go to /domains/manage
- Verify the Renew button only appears when the user can actually renew the specific domain
- Verify the Renew now button works as expected
